frontend/src/component/Activity/ActivityList.jsx

```diff
@@ -28,15 +29,27 @@ const ENTITY_TYPE_COMPONENT_CONSTRUCTOR = new Map([
           onClickCopyLink={onCopyLinkClicked}
           onEventClicked={onEventClicked}
           breadcrumbsList={breadcrumbsList}
+          lastModification={coreEventType}
+          lastModificationEntityType={entityType}
+          lastModificationSubEntityType={subEntityType}
         />
       )
       : (
-        <ContentWithPreviewActivity
-          activity={activity}
+        <FeedItemWithPreview
+          breadcrumbsList={breadcrumbsList}
+          commentList={activity.commentList}
+          content={activity.content}
+          eventList={activity.eventList}
           key={activity.id}
+          lastModifier={activity.newestMessage.fields.author}
+          lastModification={coreEventType}
```

Comment:
shouldn't this be named lastModificationType?

Reply:
fixed